src/leveled_codec.erl

```diff
@@ -751,17 +802,28 @@ return_proxy(Tag, ObjMetadata, InkerClone, JournalRef) ->
                         InkerClone,
                         JournalRef}}).
 
+-spec set_status(
+        add|remove, non_neg_integer()|infinity) ->
+            tomb|{active, non_neg_integer()|infinity}.
```

Comment:
I guess, another way of solving this is to add 2 functions and change it at the caller side... 
`get_status_add(TTL) -> {active, TTL} .`  and `get_status_remove() -> tomb`. But that requires a case statement at caller side to extract the IdxObj from the object spec. 
Not sure that that gives a lot, though

Reply:
Indeed.  I'm happy enough with it as is